### PR TITLE
forward-declare univalue in rpcclient + remove include in rpcserver.cpp

### PR DESCRIPTION
**Pull request by Diapolo**

### CODE REVIEW

File: src/rpcserver.cpp [hunk 1]
```diff
@@ -13,8 +13,6 @@
 #include "util.h"
 #include "utilstrencodings.h"
 
-#include <univalue.h>
```
Comment (jonasschnelli) [inline src/rpcserver.cpp]: Right. Not required, already in `rpcserver.h`.

**Comment by jgarzik** [comment]
ut ACK

File: src/rpcclient.h [hunk 2]
```diff
@@ -6,7 +6,10 @@
 #ifndef BITCOIN_RPCCLIENT_H
 #define BITCOIN_RPCCLIENT_H
 
-#include <univalue.h>
+#include <string>
+#include <vector>
+
+class UniValue;
```
Comment (jonasschnelli) [inline src/rpcclient.h]: This is not required. `rpcclient.cpp` has a `#include <univalue.h>`. Or do i miss something?

Comment (Diapolo) [inline src/rpcclient.h]: Right, but try to just remove it and look at line below ;-).

Comment (jonasschnelli) [inline src/rpcclient.h]: Nevermind. :)

**Comment by jonasschnelli** [comment]
code review ACK

**Comment by sipa** [comment]
Not a fan of forward declares, but meh.

**Comment by laanwj** [comment]
Meh, don't care

**Comment by dcousens** [comment]
Any motivation for this?

**Comment by laanwj** [comment]
Closing this, I don't think there's really a point to make this change.

**Comment by Diapolo** [comment]
Even if this is just "meh", you could have waited for an update, so the unneeded include in rpcclient.h could've been removed...

**Comment by jonasschnelli** [comment]
Agree with @laanwj: please just stop these PRs. They just distract from the important ones. If you can't leave it, please use @theuni trivial branch.

**Pull request closed without merging**